src-vue/tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "esModuleInterop": true,
-    "lib": ["ESNext", "DOM"],
+    "lib": ["ESNext", "DOM", "ES2018"],
```

Comment:
This forces _[ECMAScript 2018](https://learn.microsoft.com/en-us/openspecs/ie_standards/ms-es2018/38f64325-9d15-44c6-a5de-ec61a723b03f)_ right? Just so that I'm aware, why do we need to set this? ^^

Reply:
Visual Studio Code was showing promises' `finally` clauses as erroneous, unless I add this option.

Reply:
Huh, I mean if it works then ¯\\\_(ツ)\_/¯